### ISSUE TITLE
[bug] "Unable to load Chocolatey GUI assembly" after installing Chocolatey GUI.

### ISSUE THREAD

**Issue by hwhsu1231** [microsoft/winget-pkgs#83389]
### Brief description of your issue

After installing Chocolatey GUI and then launching it, an error message showed up:

```
Unable to load Chocolatey GUI assembly.
'ChocolateyGui.Common.Windows.Bootstrapper' 的初始設定式發生例外狀況。
```

![image](https://user-images.githubusercontent.com/71438617/195239150-5b6482e5-326f-4ee0-b368-79f2885694cc.png)

### Steps to reproduce

1. Install Chocolatey GUI by typeing winget command:

   ```cmd
   winget install Chocolatey.ChocolateyGUI
   ```

2. Click the icon to launch it.

### Expected behavior

Open the window of Chocolatey.

### Actual behavior

Show the error message.

### Environment

```shell
Windows 封裝管理員 v1.3.2691
Copyright (c) Microsoft Corporation. 著作權所有，並保留一切權利。

Windows: Windows.Desktop v10.0.22621.521
系統架構: X64
套件: Microsoft.DesktopAppInstaller v1.18.2691.0

記錄檔: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

連結
------------------------------------------------------------------------
隱私權聲明       https://aka.ms/winget-privacy
授權合約         https://aka.ms/winget-license
第三方聲明       https://aka.ms/winget-3rdPartyNotice
首頁             https://aka.ms/winget
Windows 市集條款 https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by Wrdle** [comment]
I actually still have this error at the moment with chocolatey CLI installed

![image](https://user-images.githubusercontent.com/9389213/196064153-423bf857-8df5-4930-a727-1dac08ccac4c.png)

**Comment by KattPhloxworthy** [comment]
I found that if the new version of Chocolatey GUI (2.0.0) doesn't find the version of Chocolatey wants to run with, even if choco.exe is actually present, it will throw this exception.  On one machine, I had _attempted_ (operative word here) to upgrade Chocolatey from 1.3.1 to 2.0.0, but it apparently didn't work, so when I tried to run the latest Chocolatey GUI, I would get this error.  Chocolatey ended up still being at 1.3.1.  Then, I found this page.  I tried the choco upgrade chocolatey -y at a command prompt, and it _insisted_ I had already upgraded it.  I tried doing an uninstall, but I couldn't reinstall because the command was still present on my system.  I ended up forcing the upgrade, adding -f.  THEN the new Chocolatey GUI started working.

Basically, make sure that Chocolatey itself actually upgraded.  If not, forcibly upgrade it at an administrator prompt.

**Comment by SirKlaus** [comment]
> I found that if the new version of Chocolatey GUI (2.0.0) doesn't find the version of Chocolatey wants to run with, even if choco.exe is actually present, it will throw this exception. On one machine, I had _attempted_ (operative word here) to upgrade Chocolatey from 1.3.1 to 2.0.0, but it apparently didn't work, so when I tried to run the latest Chocolatey GUI, I would get this error. Chocolatey ended up still being at 1.3.1. Then, I found this page. I tried the choco upgrade chocolatey -y at a command prompt, and it _insisted_ I had already upgraded it. I tried doing an uninstall, but I couldn't reinstall because the command was still present on my system. I ended up forcing the upgrade, adding -f. THEN the new Chocolatey GUI started working.
> 
> Basically, make sure that Chocolatey itself actually upgraded. If not, forcibly upgrade it at an administrator prompt.
> 
> I hope this helps those who are having problems after an upgrade of both Chocolatey and Chocolatey GUI.
> 
> --Katt. =^.^=

Thanks. Had the same problem. `choco upgrade chocolatey -f` fixed the problem for me.

**Comment by geomorillo** [comment]
> I found that if the new version of Chocolatey GUI (2.0.0) doesn't find the version of Chocolatey wants to run with, even if choco.exe is actually present, it will throw this exception. On one machine, I had _attempted_ (operative word here) to upgrade Chocolatey from 1.3.1 to 2.0.0, but it apparently didn't work, so when I tried to run the latest Chocolatey GUI, I would get this error. Chocolatey ended up still being at 1.3.1. Then, I found this page. I tried the choco upgrade chocolatey -y at a command prompt, and it _insisted_ I had already upgraded it. I tried doing an uninstall, but I couldn't reinstall because the command was still present on my system. I ended up forcing the upgrade, adding -f. THEN the new Chocolatey GUI started working.
> 
> Basically, make sure that Chocolatey itself actually upgraded. If not, forcibly upgrade it at an administrator prompt.
> 
> I hope this helps those who are having problems after an upgrade of both Chocolatey and Chocolatey GUI.
> 
> --Katt. =^.^=

choco upgrade chocolatey -f    this fixed it

**Comment by alexricher** [comment]
Ditto! 👍 I had this popup upon launching a Chocolatey GUI instance.

```
---------------------------

---------------------------
Unable to load Chocolatey GUI assembly. The type initializer for 'ChocolateyGui.Common.Windows.Bootstrapper' threw an exception.
---------------------------
OK   
---------------------------

```
I ended up doing `choco upgrade chocolatey -f` to force upgrade it, and it works as before now. Thanks!

**Comment by JulyIghor** [comment]
This issue occurs every time Windows restarts with ChocolateyGui open and installing updates.

**Comment by rodrigue1507** [comment]
thanks  KattPhloxworthy

> I found that if the new version of Chocolatey GUI (2.0.0) doesn't find the version of Chocolatey wants to run with, even if choco.exe is actually present, it will throw this exception. On one machine, I had _attempted_ (operative word here) to upgrade Chocolatey from 1.3.1 to 2.0.0, but it apparently didn't work, so when I tried to run the latest Chocolatey GUI, I would get this error. Chocolatey ended up still being at 1.3.1. Then, I found this page. I tried the choco upgrade chocolatey -y at a command prompt, and it _insisted_ I had already upgraded it. I tried doing an uninstall, but I couldn't reinstall because the command was still present on my system. I ended up forcing the upgrade, adding -f. THEN the new Chocolatey GUI started working.
> 
> Basically, make sure that Chocolatey itself actually upgraded. If not, forcibly upgrade it at an administrator prompt.
> 
> I hope this helps those who are having problems after an upgrade of both Chocolatey and Chocolatey GUI.
> 
> --Katt. =^.^=

thanks "choco upgrade chocolatey -f" works for me !!
